GeneticInheritanceGraphLibrary/tables.py

```diff
@@ -674,6 +678,102 @@ def random_match_pos(self, rng):
                             return self.MRCApos(u[0] - loc - 1, v[0] - loc - 1, False)
                 loc -= x[1] - x[0]
 
+    def plot(
+        self,
+        loc=None,
```

Comment:
I think it would be better to fully spell out this parameter, e.g. `genomic_location`?

Reply:
highlight_location is better - changed